app/code/Magento/Customer/Controller/Adminhtml/Group.php

```diff
@@ -5,71 +5,48 @@
  */
 namespace Magento\Customer\Controller\Adminhtml;
 
-use Magento\Customer\Api\Data\GroupInterfaceFactory;
-use Magento\Customer\Api\GroupRepositoryInterface;
-
 /**
- * Customer groups controller
+ * Class Group
+ *
+ * @package Magento\Customer\Controller\Adminhtml
  */
 abstract class Group extends \Magento\Backend\App\Action
 {
-    /**
-     * Authorization level of a basic admin session
-     *
-     * @see \Magento\Backend\App\Action\_isAllowed()
-     */
-    const ADMIN_RESOURCE = 'Magento_Customer::group';
 
     /**
-     * Core registry
-     *
      * @var \Magento\Framework\Registry
      */
     protected $_coreRegistry;
```

Comment:
Please make sure to add empty lines between property declarations and avoid unnecessary empty lines (including the first and last lines of class/function bodies) as that will fail the static tests build

Reply:
@sivaschenko thanks for the tip